### ISSUE TITLE
Close the pop-up when bookmark is saved

### ISSUE THREAD

**Issue by sallymaria** [Rami114/pinboard#5]
Thank you for such a quick switch to the pop-up window, it's a great improvement. It would be even better if it didn't leave an empty pop-up that has to be closed manually. I know the tab had the same problem, but it would be nice not to have to worry about it.

**Comment by Rami114** [comment]
Yep, known issue. I'll see what I can do, it might need some kind of script injection sadly as adding 'jump=close' to the URL does not work like it does on the read later.

**Comment by gfarrell** [comment]
@Rami114 anything come of this?
